translations/construction-project/de.yaml

```diff
@@ -309,11 +310,12 @@ ember-gwr:
               zw: Simbabwe
             countryNameShortPlaceholder: Wählen Sie ein Land...
 
-      typeOfConstructionProject: Bauprojekttyp
-      typeOfConstructionProjectPlaceholder: Wählen Sie einen Bauprojekttyp...
+      typeOfConstructionProject: Art der Bauwerke
+      typeOfConstructionProjectPlaceholder: Wählen Sie einen Art der Bauwerke...
```

Comment:
Changes for French are missing I think

Reply:
Is the French translation for the type of construction project correct as in the MK 4.2?

Reply:
Updated to be the same